ocs_ci/utility/ocs_build.py

```diff
@@ -0,0 +1,66 @@
+"""
+This module is used to return latest OCS internal build for specified OCS
+version.
+"""
+
+import argparse
+import os
+
+from ocs_ci.framework import config
+from ocs_ci.framework.main import load_config
+from ocs_ci.ocs.constants import OCS_VERSION_CONF_DIR
+from ocs_ci.utility.utils import get_latest_ds_olm_tag
+
+
+def init_parser():
```

Comment:
Just minor note - I would vote for some different name, like `arg_parser` or something like that.